src/Reflection/ReflectionClass.php

```diff
@@ -685,8 +690,8 @@ function (ReflectionClass $trait) use ($filter) {
 
         return array_filter(
             $this->cachedProperties,
-            function (ReflectionProperty $property) use ($filter) {
-                return $filter & $property->getModifiers();
+            function (ReflectionProperty $property) use ($filter) : bool {
```

Comment:
This should be founded with CS :(

Reply:
It's difficult. There are no annotations.

Reply:
We're basically missing a `mixed` return type (in the language) in order to have CS here too...